website/content/docs/developer-documentation.md

```diff
@@ -128,3 +128,52 @@ will then kick in and confirm your request.
 the benchmarks for it. Run it to execute the benchmark suite locally, but keep
 in mind that you must make sure your machine is stable and has been tweaked to
 get stable results.
+
+## Debugging
+
+When analysing a bug, it is often infeasible to rely only on tests. It might be
+more convenient to run Bloop and attach a debugger instead.
+
+### Running Bloop server from sbt
+
+To run the server directly from sbt:
+
+```sh
+$ sbt
+> frontend/runMain bloop.Server
+```
+
+If you want to be able to stop the server with <kbd>Ctrl</kbd><kbd>C</kbd>
```

Comment:
What about we add it to the build directly and just tell people they can do `CTRL-C` to kill it?

Reply:
I was actually thinking about that, I guess we might. Usually I add it to my global `~/.sbt/` because I want to be able to cancel any sbt task in any project. Will do.